.github/workflows/CI-python.yml

```diff
@@ -30,11 +28,6 @@ jobs:
         with:
           python-version: ${{ matrix.pythonVersion }}
 
-      - name: Use Node.js ${{ env.node-version }}
```

Comment:
looks like it's not needed since we removed raiwidgets from here

Reply:
yes.